.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,39 @@
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: #enhancement
+body:
+  - type: textarea
+    id: formulation
+    attributes:
+      label: Is your feature request related to a problem? Please describe
+      description: A clear and concise description of what the problem is
+      placeholder: |
+        I'm always frustrated when [...]
```

Comment:
Please change to positive (non-negative) vibes like `I had a situation, when [...]`

Reply:
@PeyTy done